src/components/pages/Topic.jsx

```diff
@@ -0,0 +1,13 @@
+import Header from "../Header.jsx";
+import Nav from "../Nav.jsx";
+import TopicCards from "../topic_components/TopicCards.jsx";
+
+function Topic() {
+    return <div>
+        <Header/>
```

Comment:
If you have `header` and `nav` in `app`, outside your `routes`, you won't have to repeat the import across components